### PR TITLE
English homework1 has been done.

### PR DESCRIPTION
**Pull request by sergey-ivanenko**

### CODE REVIEW

**Review by GrandCkay — DISMISSED** [review]

**Review by denisnaumovets — DISMISSED** [review]

File: homework/english/sergey_ivanenko/homework1.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+1.2 Choose the right option
+
+1. He usually has breakfast at 6 a.m.
+
+2. Anyone who (smokes / is smoking) 20 cigarettes a day can expect to have health problems.
```
Comment (danikkrapivnitskiy) [inline homework/english/sergey_ivanenko/homework1.md]: smokes "usually"
- [] add right answer

**Review by danikkrapivnitskiy — CHANGES_REQUESTED** [review]
see notes

File: homework/english/sergey_ivanenko/homework1.md (hunk 1 above)
Comment (sergey-ivanenko) [reply homework/english/sergey_ivanenko/homework1.md]: done.

**Review by GrandCkay — APPROVED** [review]

**Review by dk-t — APPROVED** [review]

**Review by danikkrapivnitskiy — APPROVED** [review]
ftm

**Pull request closed without merging**